spec/rails_helper.rb

```diff
@@ -1,28 +1,12 @@
-# frozen_string_literal: true
-
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 require 'spec_helper'
 ENV['RAILS_ENV'] ||= 'test'
-require File.expand_path('../config/environment', __dir__)
+require_relative '../config/environment'
 # Prevent database truncation if the environment is production
-abort('The Rails environment is running in production mode!') if Rails.env.production?
+abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
-# Add additional requires below this line. Rails is not loaded until this point!
+require 'support/factory_bot'
```

Comment:
to jest podejrzane, bo L#22 to laduje. Ta l#22 tez powinna byc tez poza blokiem RSpec.configure